### PR TITLE
feat: added landing page

### PR DESCRIPTION
**Pull request by techieshubham30**

### CODE REVIEW

**Review by Varu98 — APPROVED** [review]

**Review by siddhigate — APPROVED** [review]
Good to go. UI looks nice!

File: style.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+.shop-now-btn:hover {
+    background-color: var(--light-black);
+}
+
+.banner-container {
+    display: flex;
+    gap: 10rem;
+    width: 80%;
+    margin: 0 auto;
+    margin-bottom: 5rem;
```
Comment (gopalbbharadva) [inline style.css:49]: Improvement: you might missed it, redundancy of margin.

**Review by gopalbbharadva — APPROVED** [review]

File: style.css [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    position: relative;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.6);
+
+}
+
+.banner img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+}
```
Comment (BelideNikhil) [inline style.css:66]: this is not needed I guess as you already declared in globally

**Review by BelideNikhil — APPROVED** [review]
Nice UI.
Good to go!

File: style.css (hunk 1 above)
Comment (techieshubham30) [reply style.css:49]: Thanks for pointing it out, didn't notice.

File: style.css (hunk 2 above)
Comment (techieshubham30) [reply style.css:66]: Thanks for pointing it out, forgot that I have declared it globally.

**Pull request merged**